c++/include/orc/Exceptions.hh

```diff
@@ -28,8 +28,8 @@ namespace orc {
 
   class NotImplementedYet : public std::logic_error {
    public:
-    explicit NotImplementedYet(const std::string& what_arg);
-    explicit NotImplementedYet(const char* what_arg);
+    explicit NotImplementedYet(const std::string& whatArg);
+    explicit NotImplementedYet(const char* whatArg);
```

Comment:
Do we consider using snake case instead of mixed case (camel case) for variable names? Impala C++ Code follows a modified version of the Google C++ Style Guide at:
https://google.github.io/styleguide/cppguide.html#Variable_Names

I think either way is ok. Just remembered that when I first read the ORC C++ code, they look like Java code. :)

Reply:
There are lots of public/protected variables using camel case, if we change them into snake case, the library codes would be not backwards compatible. Backwards compatibility is important, I think.

Reply:
+1 for keeping the compatibility.

Reply:
I see. That makes sense to me.